packages/cozy-sharing/src/queries/queries.js

```diff
@@ -1,7 +1,21 @@
-import { Q } from 'cozy-client'
+import { Q, fetchPolicies } from 'cozy-client'
 
 import { Contact, Group } from '../models'
 
+const DEFAULT_CACHE_TIMEOUT_QUERIES = 9 * 60 * 1000
```

Comment:
Why this value? I assume the `9 * 60` computation means something?
I'm also wondering if we should not expose such default value from cozy-client, so apps/libs don't have to come up with a new value every time

Reply:
Same, this is not really the scope of this PR so we can move on without doing anything but: 

I assume that this value is copy pasted from Drive because Drive has to manipule `io.cozy.files` with `thumbnails` links only valid for 10 minutes. So we cache the value only for 9 minutes (9 * 60 * 1000). 

Maybe cozy-client should expose default cache value for a few doctypes + unlimited cache. https://github.com/cozy/cozy-client/issues/1253